README.md

```diff
@@ -26,57 +26,26 @@ Differentiating features include:
 [YAML]: http://yaml.org
 [Prometheus]: http://prometheus.io
 
-## Architecture
+## Why Chihaya?
 
-### Diagram
+Chihaya is built for developers who need to deploy a BitTorrent tracker, but have additional requirements than general purpose public trackers like those operated by [OpenBitTorrent].
```

Comment:
I think the fact that this sounded wrong to a non-native speaker is reason enough to reword it.